eng/pipelines/swagger-prettier-check.yml

```diff
@@ -10,6 +10,10 @@ jobs:
   - template: /eng/pipelines/templates/variables/globals.yml
 
   steps:
+  - checkout: self
+    # Only need head commit
```

Comment:
Don't you need 2 commits to get the file diff?

Reply:
https://dev.azure.com/azure-sdk/public/_build/results?buildId=4632068&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=0f4aa080-c997-58ca-481c-4930865ae0f8&l=12

Looks like it failed but didn't cause the pipeline to actually fail. Seems we are missing some error handling.

Reply:
I didn't know this check needed a diff.  I will update this PR to use depth:2, then open another issue to improve error handling in PrettierCheck.

Reply:
[Swagger PrettierCheck] Passes silently if "git diff" command fails (https://github.com/Azure/azure-rest-api-specs/issues/33102)